content_sync/pipelines/definitions/concourse/site-pipeline.yml

```diff
@@ -391,7 +414,7 @@ jobs:
                   mv ./content/static_resources/*.html ./static/static_resources
                 fi
                 touch ./content/static_resources/_index.md
-                aws s3((cli-endpoint-url)) sync s3://((web-bucket))/static_shared ./static/static_shared --exclude *.js.map
+                mv ../build-artifacts/static_shared ./static/static_shared
```

Comment:
I actually did test the single course pipelines the way you mentioned. The repeated directory might have got missed because I am using IDE to open the downloaded course, and by default it opens sub-directory if there is already one sub-directory. So it would looked something like this,
- static_shared/static_shared
  - js
  - css
  
  I will make the fix for this.